pkg/version/apbversion.go

```diff
@@ -0,0 +1,4 @@
+package version
+
+var MinAPBVersion = "1.0"
```

Comment:
I think that we may want these to be `const` rather than `var`.
Also, anything that is exported from the package needs a comment.